src/auth/progress.auth.form.js

```diff
@@ -118,7 +118,10 @@ limitations under the License.
                 xhr.onreadystatechange = function () {
                     if (xhr.readyState === 4) {
                         // process the response from the Web application
-                        that._processLogoutResult(xhr, deferred);
+                        try {
+                            that._processLogoutResult(xhr, deferred);
+                        } catch (e) {
```

Comment:
Do we always want to ignore the exceptions returned?

Reply:
We have taken a safe approach. Since there is no existing exception handling, we not sure what to do in case of exception, except ignoring it. I think it is fine or we could log it. We have to handle it, otherwise it would crash the node server. In one case where exceptions are actually handled, we followed the same way.

Reply:
Thank you for the reply.
We certainly want to prevent the crash and handling the exception is a way to do it.
However, it would also be important to identify the root cause so that the issue can be resolved there.
Handling the exception would prevent the crash but other behavior may be happening if the root cause is not addressed.

Reply:
I agree that we should also try and fix the root cause. However this fix is more about JS exception handling. In general it is a bad practice to leave callback handlers throwing JS exceptions. 

We have determined a cause in one the callback scenarios. It is do with the way the library is used in node environment serving multiple concurrent requests. Slightly complex to reproduce (depends on request timing), but nevertheless reproducible.